default/scripting/ship_hulls/monster/SH_IMMOBILE_FACTOR.focs.txt

```diff
@@ -15,7 +15,7 @@ Hull
     Unproducible
     location = All
     effectsgroups = [
-        [[GOOD_VISION]]
+        [[UNOWNED_OWNED_VISION_BONUS(WEAK,15,10)]]
```

Comment:
Can't have value 15, detection strengths need to always be a multiple of 10, stealth values then fit between them (normally ending in 5) to ensure there's never doubt about whether something is higher or lower.